packages/forms/src/Components/DateTimePicker.php

```diff
@@ -38,6 +38,8 @@ class DateTimePicker extends Field
 
     protected string | Closure | null $timezone = null;
 
+    protected bool | string | Closure | null $icon = null;
```

Comment:
`bool` should come after `string` in this case.

Reply:
No worries I appreciate your thoroughness

Reply:
Glad! 😄